src/ast/codegen_llvm.cpp

```diff
@@ -2156,9 +2157,12 @@ void CodegenLLVM::generateProbe(Probe &probe,
   }
   createRet();
 
+  if (dummy)
+    func->eraseFromParent();
+
```

Comment:
Why not like this (and without the `... && !dummy` below)?
```suggestion
  if (dummy)
  {
    func->eraseFromParent();
    return;
  }
```

Reply:
Yeah good point